pythran/analyses/aliases.py

```diff
@@ -207,6 +207,10 @@ def visit_Call(self, node):
             for value in return_alias:
                 if isinstance(value, NewMem):
                     all_aliases.add(value)
+                elif isinstance(value, ContainerOf):
+                    all_aliases.add(value)
+                if isinstance(value, (ast.FunctionDef, Intrinsic)):
+                    all_aliases.add(value)
```

Comment:
I thought so first, then I decided that handling may change in the
future. But you're right, the code will be cleaner.